Articles/Blog/ReferencesInMarkdownHybridApproach.md

```diff
@@ -39,10 +39,9 @@ at the end of the markdown file, of the form
   - **Note**: The post-processing script renumbers them 1...N
 - `URL`: (**REQUIRED**) The URL for the reference
 - `TIT`: (*optional*) A title for the reference (appears as tool-tip during hover)
-- `BIB`: (*optional*) Full bibliographic information for the reference
-  - Here, we are using the `{BIB}` field, embedded between `{` and `}` in the
-    [link's title](https://www.markdownguide.org/basic-syntax#adding-titles) text
-    as a notational *extension*.
+- `{BIB}`: (*optional*) Full bibliographic information for the reference
+  - Here, we are using the `{BIB}` field, embedded in the [link's title](https://www.markdownguide.org/basic-syntax#adding-titles) text as a notational *extension*.
+- **Note**: Be sure whatever editor you are using is not inadvertently sneaking in [*smart quotes* or *curly quotes*](https://practicaltypography.com/straight-and-curly-quotes.html) and that you are using only *straight* quotes.
```

Comment:
Yea, you have to be very careful when doing copy and paste from some Word document on web browser into an editor that copies these Unicode chars over.

Actually, a Markdown linter not look for these non-*straight* quotes and warn you about them?  I can't seem to find a rule like this at:

* https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md

Seems like we might need to write our own checker for that.  (Would not be hard to do that in a simple Python script.)

Reply:
> you have to be very careful when doing copy and paste

Good point. I added an additional note about that